Casks/p/plugdata.rb

```diff
@@ -7,6 +7,11 @@
   desc "Plugin wrapper for PureData"
   homepage "https://github.com/timothyschoen/PlugData"
 
+  livecheck do
+    url :stable
```

Comment:
I think we have to use 

```suggestion
    url :url
```

Reply:
👍